frontend/src/views/web/challenge/my-submission.html

```diff
@@ -42,27 +42,34 @@
                 </div>
                 <div class=" ev-card-body exist-team-card">
                     <div class="row">
-                        <div class="col s12">
+                        <div class="col s12 all-submission-table-scroll">
```

Comment:
Make a separate div for this class inside col div. Don't use own classes with grid related classes like col, m2 etc.